### PR TITLE
Look for open closed state first

### LINKED ISSUE #487: Gate (DiscreteGateWithPedestrianPosition) does not report static status
### Did you read the instructions?

- [X] I have read the [README](https://github.com/iMicknl/ha-tahoma/blob/master/README.md), including the [advanced debugging](https://github.com/iMicknl/ha-tahoma/blob/master/README.md#advanced) section.

### The problem

Hi there, since version 1.5.0 the gate does not report CLOSE or OPEN status, only statuses OPENING or CLOSING are reported while the gate is moving and after that status is switched back to UNKNOWN (for both OPEN or CLOSE gate). Thanks for checking that.

### What version of this integration (ha-tahoma) has the issue?

2.5.1

### What version of Home Assistant Core has the issue?

2021.7.2

### Which gateway / hub do you use?

Somfy TaHoma

### Device model

DiscreteGateOpenerIOComponent

### Device type

io:DiscreteGateOpenerIOComponent

### Additional information

History log:
Became unavailable
12:04:09 AM - 14 minutes ago
Is closing by Miroslav Jakovec
12:03:23 AM - 15 minutes ago
Became unavailable
12:03:14 AM - 15 minutes ago
Is opening by Miroslav Jakovec
12:02:28 AM - 16 minutes ago

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by tetienne**
Fix #487

<a href="https://gitpod.io/#https://github.com/iMicknl/ha-tahoma/pull/658"></a>

### CODE REVIEW

File: custom_components/tahoma/cover_devices/tahoma_cover.py [hunk 1]
```diff
@@ -103,20 +102,20 @@ async def async_set_cover_tilt_position(self, **kwargs):
...
-        if self.current_cover_position is not None:
-            return self.current_cover_position == 0
-
         state = self.executor.select_state(
             CORE_OPEN_CLOSED_STATE,
             CORE_SLATS_OPEN_CLOSED_STATE,
             CORE_OPEN_CLOSED_PARTIAL_STATE,
             CORE_OPEN_CLOSED_PEDESTRIAN_STATE,
             CORE_OPEN_CLOSED_UNKNOWN_STATE,
-            MYFOX_SHUTTER_STATUS_STATE,
```
Comment (iMicknl) [inline custom_components/tahoma/cover_devices/tahoma_cover.py:115]: Why did you remove this one? If I recall correctly, we use this for the Camera Shutter.

**Comment by tetienne** [comment]
@iMicknl Can you please review and merge this PR ?

**Review by iMicknl — APPROVED** [review]
